README.md

```diff
@@ -36,8 +36,18 @@ existing projects (incl. freestyle) to run on Kubernetes without changing job de
 Kubernetes builds should use the podTemplate step as shown in the example snippets [here](https://github.com/jenkinsci/kubernetes-plugin/pull/707)
 
 Fill in the Kubernetes plugin configuration. In order to do that, you will open the Jenkins UI and navigate to 
-“Manage Jenkins -> Configure System -> Cloud -> Kubernetes” and enter in the ‘Kubernetes URL’ and ‘Jenkins URL’ 
-appropriately, this is unless Jenkins is running in Kubernetes in which case the defaults work. To test this connection is successful you can use the **Test Connection** button to ensure there is 
+**Manage Jenkins -> Configure System -> Cloud -> Kubernetes** and enter in the *Kubernetes URL* and *Jenkins URL*
+appropriately, this is unless Jenkins is running in Kubernetes in which case the defaults work.
+
+Supported credentials include:
+
+* Username/password
+* Secret File (kubeconfig file) (AKS)
```

Comment:
What about this is specific to AKS? I have used this method with either GKE or EKS (I do not recall which). IMHO it is the easiest method: most cluster setup tools will automatically update your `~/.kube/config`, so it is not hard to extract the relevant bits to a separate file, verify interactively something like

```bash
KUBECONFIG=/tmp/my-new-cluster.yaml kubectl get ns
```

and then upload it to Jenkins and be pretty well assured it will work the same way, unless you have some sort of authentication helper which cannot run remotely.

Reply:
Some providers rely on an authentication helpers which may not be available in the environment where you are running the Jenkins master, or rely on addititional configuration (for example if you connect to EKS, you need a valid AWS profile in your environment).

Reply:
Maybe it was GKE that I used this with?

Reply:
GKE auth relies on `gcloud` executable being available.

Reply:
Maybe I was just using temporary credentials? Cannot recall now. Anyway I think this can be merged.